objects/client.c

```diff
@@ -805,9 +805,17 @@
  * @function set_newindex_miss_handler
  */
 
+typedef enum {
+    CLIENT_MAXIMIZED_NONE = 0 << 0,
+    CLIENT_MAXIMIZED_V    = 1 << 0,
+    CLIENT_MAXIMIZED_H    = 1 << 1,
+    CLIENT_MAXIMIZED_BOTH = 1 << 2, /* V|H == BOTH, but ~(V|H) != ~(BOTH)... */
```

Comment:
@Elv13 
The comment is not true (or I do not get it).. CLIENT_MAXIMIZED_BOTH is 4, but V|H is 3 (V being 1 and H being 0)?!
So `V|H == BOTH` is not true.